project/Build.scala

```diff
@@ -169,9 +171,9 @@ object Build {
     if (isRelease)
```

Comment:
this should really be a new condition like `isReleaseStable`?

Reply:
Something like https://github.com/scala/scala3/commit/7a7717970448c64b12f9d89fcd34b71a9354cd2d?

Reply:
https://github.com/scala/scala3/commit/7a7717970448c64b12f9d89fcd34b71a9354cd2d is just as a double guard for the CI.